packages/toast-ui.grid/src/store/helper/tree.ts

```diff
@@ -23,12 +24,18 @@ interface TreeDataCreationOption {
   disabled?: boolean;
 }
 
-let treeRowKey = -1;
+interface TreeRowKeyMap {
```

Comment:
다른 프러퍼티 추가가 없다면 `Record`를 써도 되겠네요. 단순 의견입니다.
```suggestion
type TreeRowKeyMap = Record<number, number>;
```

Reply:
인터페이스 타입이 어떠한 구조인지 더 눈에 들어오는 것 같아 이 부분은 유지하겠습니다~!ㅎㅎ